examples/graceful_shutdown.py

```diff
@@ -5,46 +5,51 @@
 from contextlib import suppress
 from datetime import datetime
 from functools import partial
+from typing import Any, Callable, Dict, Optional
 
 from aiohttp import web
 
 from aiohttp_sse import EventSourceResponse, sse_response
 
-streams = web.AppKey("streams", weakref.WeakSet[EventSourceResponse])
-worker = web.AppKey("worker", asyncio.Task[None])
```

Comment:
If subscription isn't supported on 3.8, just use a str to make it lazily evaluated:
`"asyncio.Task[None]"`

Reply:
I suppose this is what you meant in your other question.

Reply:
Python 3.11:
`error: Argument 2 to "AppKey" has incompatible type "str"; expected "Optional[Type[Never]]"  [arg-type]`

Reply:
You can use the alternative syntax, but maybe do it conditionally, so we drop it later:
```
if sys.version_info > (3, 8):
    worker_key = web.AppKey("worker_key", asyncio.Task[None])
else:
    worker_key = web.AppKey["asyncio.Task[None]"]("worker_key")
```

Reply:
Wait, forget that. This shouldn't be relevant for the examples. We can just use the latest features. There's no need to support Python 3.8 in the examples.